doc/api/errors.md

```diff
@@ -2118,6 +2118,14 @@ urlSearchParams.has.call(buf, 'foo');
 
 ### `ERR_INVALID_TRANSFER_OBJECT`
 
+<!-- YAML
+removed: REPLACEME
```

Comment:
We need to move this to the legacy error codes section if it’s no longer in use.

Reply:
Thank you for pointing this out. Updated.